src/isolate.rs

```diff
@@ -284,8 +285,6 @@ extern "C" fn pre_dispatch(
       // Set the synchronous response, the value returned from isolate.send().
       isolate.respond(req_id, buf);
```

Comment:
This respond is only called if buffer has some length, does it mean that metrics won't be updated if there is no response? Or empty response shouldn't happen in the first place?

Reply:
Ya, sounds like a bug. There are some responses with zero length - like writeFile().